.github/config.yml

```diff
@@ -17,7 +17,11 @@ newPRWelcomeComment: >
 firstPRMergeComment: >
   Thanks for your contribution to the Layer5 community! :tada:
    
-  ![Congrats!](https://raw.githubusercontent.com/meshery/meshery/master/.github/assets/images/Layer5-celebration.png)
+  <picture>
+  <source media="(prefers-color-scheme: dark)" srcset="https://raw.githubusercontent.com/anshgoyalevil/meshery/fb071249e9ecde9d936f8459c6e3ea626b22bb19/.github/assets/images/Meshery-celebration-Dark.svg">
+  <source media="(prefers-color-scheme: light)" srcset="https://raw.githubusercontent.com/anshgoyalevil/meshery/fb071249e9ecde9d936f8459c6e3ea626b22bb19/.github/assets/images/Meshery-celebration-Light.svg">
```

Comment:
@anshgoyalevil this should be at master, not a specific commit. // @ShivangShandilya @goldfishdolphin @vishalvivekm

Reply:
@leecalcote 

Apologies for that Sir.

I have created a sister PR https://github.com/meshery/meshery/pull/7586